### DISCUSSION TITLE
SPM documentation

### DISCUSSION THREAD

**Discussion by JuanCarlosZapata666** [pybamm-team/PyBaMM#3812] [General]
Hi everyone,
I was hoping if you guys know where I can find any document with the details of the SPM implementation in pybamm... Particularly, I would like to know how the negative/positive electrode reaction overpotential is implemented.
Thanks in advance.
Juan

**Comment by brosaplanella** [comment]
The Single Particle Model is defined as in the article by [Marquis et al (2019)](https://iopscience.iop.org/article/10.1149/2.0341915jes). For more details on each model you can check the documentation: https://docs.pybamm.org/en/stable/source/api/models/lithium_ion/spm.html
